psi4/driver/procrouting/empirical_dispersion.py

```diff
@@ -35,24 +34,50 @@
 from psi4.driver import p4util
 
 
+def get_default_dashparams(dtype):
```

Comment:
I've added this so that each dispersion type will return sensible defaults. The downside is that it might hide poorly defined dashparams in other parts of the code.